index.less

```diff
@@ -461,6 +461,10 @@
 }
 
 .meta {
+  & .class {
+    color: @constant;
+  }
+
```

Comment:
This unintentionally highlights the entire meta.class scope in OOP languages such as Java, C#, and D as a constant. (In seti-syntax, this is a rather off-putting red.) In pull request #28 I removed this selector.